core/templates/dev/head/profile/Profile.js

```diff
@@ -31,22 +31,83 @@ oppia.controller('Profile', [
     $rootScope.loadingMessage = 'Loading';
     $http.get(profileDataUrl).success(function(data) {
       $rootScope.loadingMessage = '';
+      $scope.username = {
```

Comment:
It's a bit weird to encode the username like this. I realize you're following the pattern for stats, but this isn't a stat and it's not displayed by one.

Instead, I suggest just saying `$scope.username = data.username;` here, and referring to `<[username]>` or `username` as appropriate in the UI.